nnpdf_data/nnpdf_data/new_commondata/PHENIX_1JET_200GEV_ALL/metadata.yaml

```diff
@@ -0,0 +1,52 @@
+setname: "PHENIX_1JET_200GEV_ALL"
+
+nnpdf_metadata:
+  experiment: "PHENIX"
+  nnpdf31_process: "JET_POL"
+
+iNSPIRE:
+  url: "https://inspirehep.net/literature/870912"
+hepdata:
+  url: "https://www.hepdata.net/record/ins870912"
+  version: 1
+
+version: 1
+version_comment: "Initial implementation"
+
+implemented_observables:
+  - observable:
+      {
+        description: "ALL w.r.t. pT",
+        label: '$\frac{d^2\sigma}{dq^{2}d\langle pT \rangle}$',
+        units: "$pb/GeV^3$",
+      }
+    observable_name: ALL
+    process_type: JET_POL
```

Comment:
```suggestion
    process_type: JET
```

Reply:
No, this really has to be `JET_POL`. The `_POL` suffix is needed to construct the MC pseudodata. Basically, anything suffixed with `_POL` is not required to have positive pseudodata.

Reply:
ah okay, fine then.